.node-version

```diff
@@ -0,0 +1 @@
+12
```

Comment:
technically 12.12.0 is what we updated to, but I assume this implies "any version within major version 12"?

Reply:
direnv's behavior is to use the highest matching version. I can make this more specific